modules/azure/billing-alert-functions/billing-function-apps/main.tf

```diff
@@ -0,0 +1,107 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.96.0"
+    }
+  }
+}
+
+resource "azurerm_app_service_plan" "main" {
+  name                = "plan-${var.func_identifier}-${var.env}-${var.location}"
```

Comment:
we removed the `env` naming convention from resources created in the foundation/core module because it is not a traditional environment such as "prod", "dev", "staging" etc. Rather than using a var here you can just use "core"